UPGRADE-3.0.md

```diff
@@ -1,6 +1,35 @@
 UPGRADE FROM 2.x to 3.0
```

Comment:
does it even make sense to have this title?
I mean: we do not support direct upgrade from 2.x to 3.0, when x < 8.